src/main/java/relay/entities/AttendanceCode.java

```diff
@@ -0,0 +1,22 @@
+package relay.entities;
+
+public class AttendanceCode {
+    private String code;
+    private String qrCodeImage;
+
+    public AttendanceCode(String code) {
+        this.code = code;
+    }
+
+    public void generateQRCode() {
+        // API call to generate QR code
```

Comment:
Can we just make the API while we're at it? Use this: [https://goqr.me/api/doc/create-qr-code/](https://goqr.me/api/doc/create-qr-code/). And I think we can pass the data as something like `https://relay.vercel.app/students?code=jf79034` where `jf79034` is the code passed into the class. Make the size of the image 100x100 pixels. See the docs.